presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java

```diff
@@ -734,7 +736,7 @@ public VerifierConfig setShadowWrites(boolean shadowWrites)
 
     public QualifiedName getShadowTestTablePrefix()
     {
-        return QualifiedName.of(Splitter.on(".").splitToList(shadowTestTablePrefix));
+        return QualifiedName.of(Splitter.on(".").splitToList(shadowTestTablePrefix).stream().map(Identifier::new).collect(Collectors.toList()));
```

Comment:
This can be changed to `Arrays.stream(fullName.split("\\.")).xxx` like the calls in TestSqlParser and formatted across multiple lines to make it more readable.